include/cuco/detail/static_multimap/device_view_impl.inl

```diff
@@ -1002,6 +1002,7 @@ class static_multimap<Key, Value, Scope, Allocator, ProbeSequence>::device_view_
           flushing_cg, *flushing_cg_counter, output_buffer, num_matches, output_begin);
         // First lane reset warp-level counter
         if (flushing_cg.thread_rank() == 0) { *flushing_cg_counter = 0; }
+        flushing_cg.sync();
```

Comment:
```suggestion
```
This is redundant since `any` would sync internally
https://github.com/NVIDIA/cuCollections/blob/88ff1e4eb7bbdd5a0d1c9025548e1451d8128a93/include/cuco/detail/static_multimap/device_view_impl.inl#L951

Reply:
As below, at least at the warp level (`__any_sync`), the intrinsic doesn't provide any memory ordering guarantees per https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#warp-vote-functions